pkgs/applications/editors/vis/default.nix

```diff
@@ -33,11 +35,10 @@ stdenv.mkDerivation rec {
       --prefix VIS_PATH : "$out/share/vis"
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "A vim like editor";
     homepage = http://github.com/martanne/vis;
-    license = stdenv.lib.licenses.isc;
-    maintainers = [ stdenv.lib.maintainers.vrthra ];
+    license = licenses.isc;
+    maintainers = [ maintainers.vrthra ];
```

Comment:
I usually avoid listing oneself as a maintainer... If I'll find myself updating it again in a couple of month, I'll reconsider :)